eoscostarica/gGoods
# Avatar maker refactor

### Avatar maker refactor

### What does this PR do?

- Resolve #231 #116 

### Steps to test

1. run `make run`
2. go to localhost:3000
3. check the avatar maker component (in mobile and desktop version)
4. go to game and see the user's nfts loaded

#### CheckList

- [] Follow proper Markdown format
- [] The content is adequate
- [] The content is available in both english and spanish
- [] I Ran a spell check
